src/Microsoft.TestPlatform.CrossPlatEngine/Client/InProcessProxyDiscoveryManager.cs

```diff
@@ -113,4 +113,9 @@ private void InitializeExtensions(IEnumerable<string> sources)
         // it will use TestPluginCache of vstest.console
         _discoveryManager.Initialize(Enumerable.Empty<string>(), null);
     }
+
+    public void InitializeDiscovery(DiscoveryCriteria discoveryCriteria, ITestDiscoveryEventsHandler2 eventHandler, bool skipDefaultAdapters)
+    {
+        throw new NotImplementedException();
```

Comment:
Could you add a comment about why it's not implemented?

Reply:
Fixed.